packages/sparql-proxy/test/support/workerDouble.js

```diff
@@ -0,0 +1,12 @@
+import * as url from 'node:url'
+import sinon from 'sinon'
+import rdf from '@zazuko/env-node'
+import fetch from '../../lib/fetchServiceDescription.js'
+
+const serviceDescriptionPath = url.fileURLToPath(new URL('./serviceDescription.ttl', import.meta.url))
+
+fetch.fetchServiceDescription = sinon.stub().callsFake(async () => {
+  return rdf.dataset().import(rdf.fromFile(serviceDescriptionPath))
+})
```

Comment:
The mocking is done on a tiny helper module to avoid `fetch` completely.

Now that I look at this, `sinon` is not necessary here but I use it anyway to check that the default fetch is not used at all

Reply:
I wonder how manually crafted test doubles would compare to using `sinon`.  Probably more verbose?

Reply:
I find manual test doubles always a bit clunky. sinon is super clean, especially combined with `sinon-chai`